README.md

```diff
@@ -7,6 +7,12 @@ The Changeset comment automatically gets an attachment to the comment showing th
 Once the changes are uploaded, the Note that was referenced gets a comment showing the URL of the Changeset and gets resolved.
 
 While this plugin is not yet published to the JOSM repository, use this description of how to add the plugin to JOSM manually.
+## How to install this plugin
+
+Download [notesolver.jar](https://github.com/kmpoppe/noteSolver/releases/download/0.3.1/notesolver.jar) and copy it to "plugins" inside the [user-data directory](https://josm.openstreetmap.de/wiki/Help/Preferences#JOSMpreferencedatacachedirectories).
```

Comment:
The general README should contain a link to the latest release rather than a specific one, right?
```suggestion
Download [notesolver.jar](https://github.com/kmpoppe/noteSolver/releases/latest/download/notesolver.jar) and copy it to "plugins" inside the [user-data directory](https://josm.openstreetmap.de/wiki/Help/Preferences#JOSMpreferencedatacachedirectories).
```